client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -105,32 +113,65 @@ class PublishingTools extends Component {
 		const formPending = this.isFormPending();
 
 		return (
-			<FormFieldset>
-				<SupportInfo
-					text={ translate(
-						'Allows you to publish new posts by sending an email to a special address.'
-					) }
-					link={
-						isAtomic
-							? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
-							: 'https://jetpack.com/support/post-by-email/'
-					}
-					privacyLink={ ! isAtomic }
-				/>
-				<JetpackModuleToggle
-					siteId={ selectedSiteId }
-					moduleSlug="post-by-email"
-					label={ translate( 'Publish posts by sending an email' ) }
-					disabled={ formPending || moduleUnavailable }
-				/>
+			<>
+				<FormFieldset>
+					<SupportInfo
```

Comment:
```suggestion
			<>
				<QueryJetpackConnection siteId={ selectedSiteId } />

				<FormFieldset>
					<SupportInfo
```
Let's move the `QueryJetpackConnection` component from the `render` method here. There is no reason to query anything Jetpack-related if we aren't rendering the Post by Email settings.